index.html

```diff
@@ -78,6 +78,9 @@ <h2>NS</h2>
         
      </div>
     </div>
+<footer>
+    <h5>Created by: Steve and Anita</h5>
+</footer>
```

Comment:
Great use of footer tag to help with semantic HTML! 👍

Reply:
@steveWDamesJr 

Thank you 🤝 

Happy Coding!